__tests__/pet.test.js

```diff
@@ -0,0 +1,102 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Dexter')).toBeInstanceOf(Object);
+    });
+});
+describe('naming pet', () => {   
+    it('sets the name property', () => {
+        const pet = new Pet('Dexter');
+        expect(pet.name).toEqual('Dexter');
+    });
+});
+describe('initial age', () => {
+    it('has a initial age of 0', () => {
+        const pet = new Pet('Dexter');
+        expect(pet.age).toEqual(0);
+    });
+});
```

Comment:
These could be grouped together a little better. Id say they are all testing different parts of the constructor, so they should be kept inside a describe block that covers all of them:
```
describe('constructor', () => {
    it('returns an object', () => {
        expect(new Pet('Dexter')).toBeInstanceOf(Object);
    });

    it('sets the name property', () => {
        const pet = new Pet('Dexter');
        expect(pet.name).toEqual('Dexter');
    });

    it('has a initial age of 0', () => {
        const pet = new Pet('Dexter');
        expect(pet.age).toEqual(0);
    });
});
```
Also added spaces between the `it` blocks to make it easier to read, which you should do with all tests